packages/wonder-blocks-core/components/clickable.js

```diff
@@ -0,0 +1,141 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import {Link} from "react-router-dom";
+import addStyle from "../util/add-style.js";
+import getClickableBehavior from "../util/get-clickable-behavior.js";
+import type {ClickableRole, State} from "./clickable-behavior.js";
+import type {StyleType} from "../util/types.js";
+
+type Props = {|
+    "aria-label": string,
```

Comment:
Can you add comments to these props so that they show up in the documentation? You can probably copy most of them from Link or Button.